src/components/header/Header.tsx

```diff
@@ -1,61 +1,88 @@
+                  alt="logo"
+                />
+              </Link>
+            </div>
+            <div className="search">
+              <input type="search" placeholder="What´s search today?" />
+            </div>
+          </nav>
+          <footer className="menu">
             <div>
-                <Link to="/">
-                    <img src="src/components/header/home.png" alt="" />
-                </Link>
+              <Link to="/">
+                <img src="src/components/header/home.png" alt="Home" />
+              </Link>
             </div>
             <div>
-                <Link to="/shopping-cart">
-                    <img src="src/components/header/shopping-cart.png" alt="" />
-                </Link>
+              <Link to="/shopping-cart">
+                <img
+                  src="src/components/header/shopping-cart.png"
+                  alt="Shopping Cart"
+                />
+              </Link>
             </div>
             <div>
-                <Link to="/access">
-                    <img src="src/components/header/user.png" alt="" />
-                </Link>
+              <Link to="/access">
+                <img src="src/components/header/user.png" alt="User" />
+              </Link>
             </div>
-        </footer>
-        </>) : ( 
+          </footer>
+        </>
+      ) : (
         <nav className="nav">
-            <div className="logo-laptop"><Link to="/"><img src="https://www.globant.com/themes/custom/globant_corp_theme/images/2019/globant-logo-dark.svg" alt="logo" /></Link></div>
-            <div className="search"><input type="search" placeholder="What´s search today?" /></div>
-            <div className="list">
-                <ul className="list-menu">
-                    <li>
-                    <Link to="/">Home</Link></li>
-                    <li><Link to="/shopping-cart">Cart</Link></li>
-                </ul>
-            </div>
-            <div className="access">
-            <Link to="/access"><button>Access</button></Link>
-            </div>
-        </nav>) 
-
-        }
+          <div className="logo-laptop">
+            <Link to="/">
+              <img
+                src="https://www.globant.com/themes/custom/globant_corp_theme/images/2019/globant-logo-dark.svg"
+                alt="Globant Logo"
+              />
+            </Link>
+          </div>
+          <div className="search">
+            <input type="search" placeholder="What´s search today?" />
+          </div>
+          <div className="list">
+            <ul className="list-menu">
+              <li>
+                <Link to="/">Home</Link>
+              </li>
+              <li>
+                <Link to="/shopping-cart">Cart</Link>
+              </li>
+            </ul>
+          </div>
+          <div className="access">
+            <Link to="/access">
+              <button>Access</button>
+            </Link>
+          </div>
+        </nav>
+      )}
```

Comment:
If this is for prettier, this shouldn't be here

Reply:
yes, i understanded. my bad